lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -1,10 +1,36 @@
 class ActiveModel::Serializer::Adapter::JsonApi < ActiveModel::Serializer::Adapter
+        # JsonApiObject: namespace for functions that build
+        # JSON API objects
+        module JsonApiObject
+          # Make JSON API top-level jsonapi member opt-in
+          # ref: http://jsonapi.org/format/#document-top-level
+          ActiveModel::Serializer.config.jsonapi_toplevel_member = false
+          ActiveModel::Serializer.config.jsonapi_version = '1.0'
+          ActiveModel::Serializer.config.jsonapi_toplevel_meta = {}
+          module JsonApi
```

Comment:
good call. my intention was that I was considering having a whole bunch of `JsonApiObjects` like `JsonApi`, `Link`, `Error`, `Meta`, `Data`, etc.